pages/what-we-do/exemplar-projects.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Exemplar projects
+layout: basic
+permalink: /what-we-do/partnerships/exemplar-projects/
+localnav: what-we-do
+gov-speak: true
```

Comment:
We usually only use gov-speak for annual-reports and other long form documents. It's documented here: http://guides.service.gov.au/design-guide/foundations/typography/index.html#headings-guidance-accordion